.buildkite/premerge.steps.yaml

```diff
@@ -22,7 +22,13 @@ common: &common
 
 steps:
   - label: "build-GDK-:windows:"
-    command: "powershell ./ci/setup-and-build-gdk.ps1"
-    <<: *common # This folds the YAML named anchor into this step. Overrides, if any, should follow, not precede. 
+    command: "powershell ./ci/setup-and-build-gdk.ps1 -target-platform Win64"
```

Comment:
Shouldn't `target-platform` be `target_platform`? Or is there some automagic character replacement going on?